sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -16,6 +16,11 @@
   /** Enable or disable ANR on Debug mode Default is disabled Used by AnrIntegration */
   private boolean anrReportInDebug = false;
 
+  /**
+   * enable the usage of Settings.Secure.ANDROID_ID to keep retro compatible with older SDK versions
+   */
+  private boolean enableSecureAndroidId;
```

Comment:
Could this option be named in such as way as to indicate a fallback (as opposed to `enable` which sounds like a feature).
Perhaps `useLegacyUserId` or something else?

Also, should this be guarded by `sendDefaultPii`? Meaning we'd need both this flag on + `sendDefaultPii`?

Reply:
agreed, naming here can be improved, `useLegacyAndroidId` maybe? the behavior is very dependent on it.
so initially I've added `sendDefaultPii` and later removed. as it's the only field to be guarded, it sounds redundant, one would need to swap 2 flags for a single behavior change.